e2e/assets/single-cluster/helm-kustomize.yaml

```diff
@@ -0,0 +1,8 @@
+kind: GitRepo
+apiVersion: fleet.cattle.io/v1alpha1
```

Comment:
nit: apiVersion and kind are flipped, I usually see apiVersion come first in manifests